levels/level-1.md

```diff
@@ -0,0 +1,40 @@
+# Brukergrensesnittutvikler, nivå 1
+
+## Tittel
+
+Brukergrensesnittutvikler / User Interface Developer
+
+## Beskrivelse
+
+Den ansatte skal som regel jobbe i prosjekt sammen med én eller flere andre brukergrensesnittutviklere, og skal få hjelp og veiledning fra disse.
+
+Det forventes deltagelse i planlegging og gjennomføring av utvikling av gjenbrukbare enkeltkomponenter og komponentsystemer.
+
+## Erfaring
+
+Relevant utdannelse på bachelornivå med svært gode resultater
+
+Kortere relevant erfaring fra prosjektoppgaver, deltidsjobber, egne prosjekter e.l.
+
+_eller_
+
+Relevant erfaring fra forrige nivå
+
+## Faglig kompetanse
+
+Grunnleggende forståelse for, og erfaring med kildekodehåndteringsverktøy som Git, SVN, Mercurial e.l.
+
+- Må kunne bruke de mest vanlige kommandoene og vite hva de gjør (f.eks. `git [clone, checkout, branch, commit, push, pull, merge]`)
```

Comment:
rebase?

Reply:
Must be able to only ever use rebase locally

Reply:
It's not among the most commonly used commands, so it's fine just knowing not to use it on any commits that has been pushed to the repository